backend/src/main/scala/cromwell/backend/standard/StandardFinalizationActor.scala

```diff
@@ -0,0 +1,36 @@
+package cromwell.backend.standard
+
+import cromwell.backend._
+import cromwell.core.CallOutputs
+import wdl4s.TaskCall
+
+trait StandardFinalizationActorParams {
+  val workflowDescriptor: BackendWorkflowDescriptor
+  val calls: Set[TaskCall]
+  val jobExecutionMap: JobExecutionMap
+  val workflowOutputs: CallOutputs
+  val initializationDataOption: Option[BackendInitializationData]
+  val configurationDescriptor: BackendConfigurationDescriptor
+}
+
+case class DefaultStandardFinalizationActorParams
+(
+  workflowDescriptor: BackendWorkflowDescriptor,
+  calls: Set[TaskCall],
+  jobExecutionMap: JobExecutionMap,
+  workflowOutputs: CallOutputs,
+  initializationDataOption: Option[BackendInitializationData],
+  configurationDescriptor: BackendConfigurationDescriptor
+) extends StandardFinalizationActorParams
+
+trait StandardFinalizationActor extends BackendWorkflowFinalizationActor {
+  /** Standard set of parameters passed to the backend. */
```

Comment:
Considering `standard` and `params` exists twice each on the following line I don't think this scaladoc adds much